modules/backend/lang/lv/lang.php

```diff
@@ -191,272 +287,365 @@
         'select_all' => 'izvēlēties visus',
         'select_none' => 'neizvēlēties nevienu',
-        'select_placeholder' => 'lūdzu izvēlieties',
+        'select_placeholder' => 'lūdzu, izvēlieties',
         'insert_row' => 'Ievietot rindu',
-        'insert_row_below' => 'Ievietot riendu zemāk',
+        'insert_row_below' => 'Ievietot rindu zem',
         'delete_row' => 'Dzēst rindu',
-        'concurrency_file_changed_title' => 'Fails tika modificēts',
-        'concurrency_file_changed_description' => "Fails, kuru labojat ir ticis modificēts no cita lietotāja puses. Jūs varat pārlādēt failu un zaudēt savas izmaiņas vai arī pārrakstīt esošo failu uz diska.",
-        'return_to_list' => 'Atgriezties uz sarakstu'
+        'concurrency_file_changed_title' => 'Fails ir ticis modificēts',
+        'concurrency_file_changed_description' => "Failu, kuru labojat, ir modificējis cits lietotājs. Jūs varat vai nu pārlādēt failu un zaudēt savas veiktās izmaiņas, vai arī pārrakstīt esošo failu uz diska.",
+        'return_to_list' => 'Atgriezties uz sarakstu',
     ],
     'recordfinder' => [
-        'find_record' => 'Atrast ierakstu'
+        'find_record' => 'Atrast ierakstu',
+        'invalid_model_class' => 'Norādītā modeļa klase ":modelClass" nav derīga ierakstu meklētājam',
+        'cancel' => 'Atcelt',
+    ],
+    'pagelist' => [
+        'page_link' => 'Lapas saite',
+        'select_page' => 'Izvēlieties lapu...',
     ],
     'relation' => [
-        'missing_config' => "Relācijām nav norādīta nekāda konfigurācija ':config'.",
-        'missing_definition' => "Relācijām nav definēts lauks ':field'.",
-        'missing_model' => "Relācijās izmantotajai klasei :class nav moduļa definīcijas.",
-        'invalid_action_single' => "Šī darbība nevar tikt veikta ar vienmoduļa relāciju.",
-        'invalid_action_multi' => "Šī darbība nevar tikt veikta ar daudzmoduļu relāciju.",
-        'help' => "Spiediet uz vienuma, lai pievienotu",
-        'related_data' => "Saistītie :name dati",
-        'add' => "Pievienot",
-        'add_selected' => "Pievienot izvēlētos",
-        'add_a_new' => "Pievienot jaunu :name",
-        'link_selected' => "Saite izvēlēta",
-        'link_a_new' => "Saistīt jaunu :name",
-        'cancel' => "Atcelt",
-        'close' => "Aizvērt",
-        'add_name' => "Pievienot :name",
-        'create' => "Izveidot",
-        'create_name' => "Izveidot :name",
-        'update' => "Atjaunot",
-        'update_name' => "Atjaunot :name",
-        'preview' => "Priekšskatīt",
-        'preview_name' => "Priekšskatīt :name",
-        'remove' => "Noņemt",
-        'remove_name' => "Noņemt :name",
-        'delete' => "Dzēst",
-        'delete_name' => "Dzēst :name",
-        'delete_confirm' => "Vai esat pārliecināts?",
-        'link' => "Saistīt",
-        'link_name' => "Saistīt :name",
-        'unlink' => "Atsaistīt",
-        'unlink_name' => "Atsaistīt :name",
-        'unlink_confirm' => "Vai esat pārliecināts?",
```

Comment:
Changes that are just swapping quote types around shouldn't be included in this PR, if we decide to normalize them at some point we'll do that for all the files in a single commit to keep the history clean. If you could revert the changes that are only swapping quote types that would be great. Thanks for all your hard work on this!

Reply:
I agree that quote swapping may be bad for the git history but one of the goals of these Latvian translation updates were also to normalize and unify translations against the English versions. So all quote changes and extra commas for the last array values etc. were inserted on purpose to unify translations with English ones, to reduce potential issues and maintenance work in future. Also the mentioned `modules/backend/lang/lv/lang.php` file has many indention changes before character `=>` because the English version have introduced new parameters/values which didn't existed in Latvian version.

I could try to revert changes but in overall that would make 2-3x more effort:
1) because I would need to compare changed string with previous version (overall I am ok with this);
2) as I mentioned, problem is not only with quotes but also with commas and spaces before `=>` - to sum it up, Latvian translations were just too outdated and didn't have many changes made in English translations. It also means that before there wasn't no normalization process for other languages  when new strings and quote changes were introduced in English files. My changes were not made to complicate or ruin history but to normalize things.
3) If now I revert quote, space, comma changes then in future someone would need to do additional extra work to normalize things back again..., and it also will clutter history.

In my opinion we just accept that translations were too outdated and many changes were needed so in the future there would be less issues when adding new translations or when big changes are made to English language files. Also my proposed changes and versions are easier to use for future translators because if both English and Latvian version contains the same quote style and indentation then it's easier to see what strings has been added or removed from English files.

Short version - I am waiting for additional feedback on this before reverting something. And if we are starting to revert something then it applies only to quotes or also to commas (for the last array values) and spaces before `=>`?

Reply:
Fair enough!